drivers/hmis/lib/form_data/default/records/ce_assessment.json

```diff
@@ -0,0 +1,50 @@
+{
+  "item": [
+    {
+      "type": "GROUP",
+      "link_id": "event-group",
+      "item": [
+        {
+          "type": "DATE",
+          "required": true,
+          "link_id": "4.19.1",
+          "text": "Assessment Date",
+          "mapping": {
+            "field_name": "assessmentDate"
+          }
+        },
+        {
+          "type": "TEXT",
+          "required": true,
+          "link_id": "4.19.2",
+          "text": "Assessment Location",
+          "mapping": { "field_name": "assessmentLocation" }
```

Comment:
>  It is recommended that a system administrator managed list is used for this
field. If such functionality doesn’t exist in the HMIS, a text box must be provided.

@eanders do you know what this location is supposed to be, is it the CoC? or some other custom managed list of locations?

Reply:
I don't know.  I believe we've seen everything from the CoC Code to the project or organization name to a specific address.  My gut would be project names...

Reply:
Huh okay. None of those really make sense to me, because you should be able to auto determine that based on the Enrollment where you're recording this. Unless the location is typically at a _different_ project? I think we leave as text for now, and ask.

Reply:
Oh, yeah, sorry, I should have mentioned that. That field is supposed to be a pick list of projects but the logic seemed thorny so I was going to circle back around on it and ping you about how we should do it after I was done with everything else.